CMakeLists.txt

```diff
@@ -36,7 +36,6 @@ FIND_PACKAGE(Boost 1.47.0 COMPONENTS chrono system REQUIRED)
 INCLUDE_DIRECTORIES(SYSTEM ${Boost_INCLUDE_DIRS})
 
 SET(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/Modules)
-FIND_PACKAGE(Rabbitmqc REQUIRED)
```

Comment:
why?

Reply:
This was done a long time ago (we are already building navitia without the package since 2013). This is because we want to link Kraken against the librabbitmq from the `thirdparty` folder.

The proper way of fixing this (eg. putting back the `FIND_PACKAGE`), would be to introduce some kind of package dependency manager (and C++ is rubbish at handling this). 

I'm just highlighting a change made long ago here :)